src/main/java/ec/com/pablorcruh/gym_management_system/domain/models/Membership.java

```diff
@@ -0,0 +1,15 @@
+package ec.com.pablorcruh.gym_management_system.domain.models;
+
+import ec.com.pablorcruh.gym_management_system.domain.enums.MembershipType;
+
+import java.math.BigInteger;
+import java.time.LocalDateTime;
+
+public class Membership {
+    private BigInteger price;
+    private LocalDateTime firstDay;
+    private LocalDateTime lastDay;
+    private MembershipType membershipType;
+    private Boolean state;
```

Comment:
I would change this to `is_active`, because state could mean anything. If you think the state could have more than two values, this should be an `enum`. Anyways, the naming should be changed to be more descriptive.